consensus/consortium/v2/consortium_test.go

```diff
@@ -2532,3 +2536,174 @@ func TestEncodeDecodeValidatorBitSet(t *testing.T) {
 		}
 	}
 }
+
+func randFieldElement() [32]byte {
+	bytes := make([]byte, 32)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		panic("failed to get random field element")
+	}
+	var r fr.Element
+	r.SetBytes(bytes)
+
+	return gokzg4844.SerializeScalar(r)
+}
+
+func randBlob() *kzg4844.Blob {
+	var blob kzg4844.Blob
+	for i := 0; i < len(blob); i += gokzg4844.SerializedScalarSize {
+		fieldElementBytes := randFieldElement()
+		copy(blob[i:i+gokzg4844.SerializedScalarSize], fieldElementBytes[:])
+	}
+	return &blob
+}
+
+func TestVerifyBlobHeader(t *testing.T) {
+	c := Consortium{
+		chainConfig: &params.ChainConfig{
+			TrippBlock: common.Big0,
+		},
+		config: &params.ConsortiumConfig{
+			EpochV2: 200,
+		},
+		isTest: true,
+	}
+	// Create 100 random txs
+	var txs []*types.Transaction
+	var blobs []kzg4844.Blob
+	var commitments []kzg4844.Commitment
+	var proofs []kzg4844.Proof
+
+	var fakeCommitments []kzg4844.Commitment
+	var fakeProofs []kzg4844.Proof
+
+	for i := 0; i < 100; i++ {
+		nBlobs, err := rand.Int(rand.Reader, big.NewInt(7))
+		if err != nil {
+			t.Fatal(err)
+		}
+		hasher := sha256.New()
+		var blobHashes []common.Hash
+		for j := 0; j < int(nBlobs.Int64()); j++ {
```

Comment:
Similarly, a separate func to create mock blob tx

Reply:
I have moved them to separate functions